api/src/main/java/org/pmiops/workbench/billing/BillingProjectBufferService.java

```diff
@@ -137,13 +140,34 @@ public void syncBillingProjectStatus() {
 
   public void cleanBillingBuffer() {
     Iterables.concat(
```

Comment:
nit: I'm finding this hard to follow, since the arguments to concat() are so involved and far apart. Why not declare two lists separately and concat them at the end? Seems like that's a win for debuggability too.

Reply:
So I think part of the issue is that I trained myself to try to read stream statements in one mental "bite" but I can't really do that here. Which may be fine. In other words, if most code is stream operations, this will become more natural.

The biggest issue reading it for me was that I couldn't see the whole statement at once on my laptop, so I have to remember context (like the iterables.concat() around everything), try to find the comma, etc.